spec/services/candidate_interface/continuous_applications/submit_application_choice_spec.rb

```diff
@@ -30,6 +30,16 @@
         end
       end
 
+      it 'does not updated submitted_at for a second application choice' do
+        travel_temporarily_to(Time.zone.local(0)) do
+          submit_application
```

Comment:
I think you have to call `submit_application` twice in this spec.

Reply:
oops – good catch. updated!